packages/nextlove/src/with-route-spec/index.ts

```diff
@@ -51,12 +51,13 @@ export const createWithRouteSpec: CreateWithRouteSpecFunction = ((
     exceptionHandlingMiddleware = withExceptionHandling({
       addOkStatus: setupParams.addOkStatus,
       exceptionHandlingOptions: {
-        shouldIncludeErrorContext: () =>
-          Boolean(
-            process.env.CI ||
-              process.env.NODE_ENV === "development" ||
-              process.env.NODE_ENV === "test"
-          ),
+        getErrorContext: (req, error) => {
+          if (process.env.NODE_ENV === "production") {
+            return {}
+          }
+
+          return error
```

Comment:
@codetheweb should we serialize this into some kind of object?

@itelo is it possible to add a test for this?

Reply:
This is what we do in a test on `nextjs-exception-middleware` and it seems to work, but yeah explicit serialization would be nice

Reply:
looking at it